palm/plugins/dbt/commands/cmd_cycle.py

```diff
@@ -53,22 +51,14 @@ def run_test():
         )
 
     def make_cmd():
-        if fast:
-            if no_seed:
-                seed_cmd = ""
-            else:
-                seed_cmd = "dbt seed --full-refresh"
-            commands = [add_select(seed_cmd), run_test(), add_persist()]
-        else:
-            if no_seed:
-                seed_cmd = ""
-            else:
-                seed_cmd = " && dbt seed --full-refresh "
-            commands = [
-                add_select("dbt clean && dbt deps" + seed_cmd),
-                run_test(),
-                add_persist(),
-            ]
+        commands = []
```

Comment:
cycle should mimic the behavior of CI as closely as possible, afaik, unless I misunderstood it's primary use

Reply:
Aside from removing clean & deps, the logic here has not changed. I think this implementation still matches what we're doing in CI with cycle_test. Is there something I'm missing?

Also per our discussion in slack yesterday, we shouldn't think about commands in palm-dbt as mimicking CI, since CI will be different for others in the community. Rather, we may want to think about ensuring our CI mimicks the decisions we make in palm-dbt